tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -16,13 +16,15 @@
     BaseCType,
     Binding,
     boolT,
+    ConstRefCType,
     CType,
     DispatcherSignature,
     intArrayRefT,
     longT,
     OptionalCType,
     symIntArrayRefT,
     SymIntT,
+    tensorT,
```

Comment:
nit: maybe copy the title of your [note] here, so it's clear what bits of the codegen we can eventually blow up when we switch NestedTensor over to symints.